src/openforms/utils/json_logic.py

```diff
@@ -49,3 +49,123 @@ def normalize_value(value: Union[dict, list, tuple, int, str]) -> JSONLogicValue
         return JsonLogicTest.from_expression(value)
 
     raise NotImplementedError(f"Unknown value type: {type(value)}")
+
+
+def find_json_logic_test_value(value: JsonLogicTest) -> str:
+
+    """returns the value inside a JsonLogicTest"""
```

Comment:
```suggestion
def find_json_logic_test_value(value: JsonLogicTest) -> str:
    """returns the value inside a JsonLogicTest"""
```
docstrings should not be separated by a blank line